### ISSUE TITLE
Backspace and scroll makes text jerk when deleting many newlines

### ISSUE THREAD

**Issue by karpelcevs** [steipete/PSPDFTextView#1]
First, thanks for fixing most of the iOS7 bugs! Unfortunately, there are still some more.

In the example project, add to the text quite a few simple newlines: `\n\n\n\n\n\n\n\n\n\n\`. It works also with letters and newlines, such as `a\na\na\na\na\n` etc.
Launch the app, tap the text.

First, ensure-visible issues:
1. Text isn't scrolled to bottom, it stays somewhere in the middle.
2. Second tap also doesn't scroll to bottom even though caret is now there.

Now, start tapping backspace. Newlines are removed, however:
1. Big text piece from above is flashing at the top of the view right after pressing backspace and disappearing again (as if scrolling up and then down again).
2. Caret is jumping somewhere up and then down again (I suppose this _is the reason_ for jerking, but it's just a guess).

**Important note**, this is reproducible only when text is added prior to editing. Adding newlines from keyboard and then removing them does not reproduce the behaviour.

I'm trying to fix it in my app myself right now. If I'm successful, will add a pull request as well.

**Comment by getaaron** [comment]
@coverback  Any luck solving this?

**Comment by karpelcevs** [comment]
@getaaron no. Sad to say, I had to go live with this bug. Investigation showed that caret tends to be in incorrect position until at least a new event cycle completes, and I decided that a dirty hack by waiting could break more than solve.

**Comment by poscar** [comment]
Also experiencing this issue :(
